src/webgpu/api/validation/encoding/queries/resolveQuerySet.spec.ts

```diff
@@ -1,13 +1,94 @@
 export const description = `
-TODO:
-- invalid GPUQuerySet
-- firstQuery and/or queryCount out of range
-- invalid destination buffer
```

Comment:
This is referring to an invalid buffer, i.e. failed during creation. See `ValidationTest.createBufferWithState` but feel free to leave this `TODO` or `.unimplemented()` in this PR.